tests/test_features/test_projection.py

```diff
@@ -238,5 +267,15 @@ def test_y_required_for_discrete_and_continuous(self, dataset):
         visualizer.fit(X, y)
 
         msg = "y is required for {} target".format(dataset)
-        with pytest.raises(YellowbrickValueError, match = msg):
-            visualizer.transform(X)
\ No newline at end of file
+        with pytest.raises(YellowbrickValueError, match=msg):
+            visualizer.transform(X)
```

Comment:
Just to confirm - most of the above is `black` formatting correct? The only thing that has been changed here is the test of the colorbar layout?

Reply:
Yep, most of it is black formatting. I think that we might have done a pretty good job with the formatting here. ; )